spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -232,7 +232,7 @@ module Medium
   end
 
   it 'does not register offense for absolute file path' do
-    allow(File).to receive(:expand_path).with('my_class_spec.rb').and_return(
+    allow(cop).to receive(:expanded_file_path).and_return(
```

Comment:
RSpec suggests a way of fixing this:
```
       #<File (class)> received :expand_path with unexpected arguments
         expected: ("my_class_spec.rb")
              got: ("/Users/pirj/source/rubocop-rspec")
        Please stub a default value first if message might be received with other args as well.
```

```suggestion
    allow(File).to receive(:expand_path).and_call_original # RuboCop's Commissioner is calling it, too
    allow(File).to receive(:expand_path).with('my_class_spec.rb').and_return(
```